dj/main/models.py

```diff
@@ -88,7 +88,6 @@ class Location(models.Model):
     default = models.BooleanField(default=True,
         help_text="Adds this loc to new Clients.")
     name = models.CharField(max_length=135,
-        default=socket.gethostname(),
```

Comment:
good catch - this was  left over from when we generally ran things local.  or something.